.travis.yml

```diff
@@ -14,4 +15,5 @@ script: vendor/bin/phpunit
 
 matrix:
   allow_failures:
+    - php: 7.0
```

Comment:
If you want to know you just have to check the concerned job: https://travis-ci.org/guzzle/streams/jobs/73407368

Could be useful for php7 compat preparation, but should not be a constraint for another fix/feature PRs.